lecture_notes/08 collections/8-map.py

```diff
@@ -0,0 +1,19 @@
+nested_dict = {'ages': {
+    'Map': {
+        'Vasia Pupkin': 19,
+        'Marcus Aurelius': 1860
+    }
+}}
+
+print(nested_dict)
+
+nested_dict['ages']['Map']['Vasia Pupkin'] = 20
+print(nested_dict)
+del nested_dict['ages']['Map']['Vasia Pupkin']
+
+print(nested_dict)
+
+print('Vasia Pupkin:', 'Vasia Pupkin' in nested_dict['ages']['Map'], 'Marcus Aurelius:', 'Marcus Aurelius' in nested_dict['ages']['Map'])
```

Comment:
- Rename `m` to better name in following block
```suggestion
m = nested_dict['ages']['Map']
print({
  'Vasia Pupkin:', 'Vasia Pupkin' in m,
  'Marcus Aurelius:', 'Marcus Aurelius' in m
})
```